include/stdx/compiler.hpp

```diff
@@ -22,12 +22,24 @@
 #endif
 #endif
 
+#ifndef USING_ATTR_NS
+#if defined(__clang__)
+#define USING_ATTR_NS using clang:
+#else
+#define USING_ATTR_NS using gnu:
+#endif
+#endif
```

Comment:
Question: should this be "clang::"?

Reply:
No... this is selecting between the clang and gnu namespaces.